query-connector/src/app/shared/DataProvider.tsx

```diff
@@ -63,6 +63,7 @@ export function DataProvider({
       <SessionProvider
         session={session}
         refetchInterval={REFRESH_INTERVAL_MINS * 60}
+        refetchOnWindowFocus={false}
```

Comment:
The app was triggering a call to the api/auth endpoint to resync the session object but the 15 mins interval should address any sync. Disabling resync on focus to save some HTTP requests.

Reply:
I know this isn't the first place where we added `REFRESH_INTERVAL_MINS`, but could we add a comment explaining how we picked 15 minutes (even if that comment is "it's arbitrary, Johanna just picked 15"? Could see a future where this figure is questioned for our compliance with some security protocol, and ideally would like to document the thought process to make sure future us has whatever context we can give them

Reply:
done!